portals/admin/src/main/webapp/source/src/app/components/Workflow/WorkflowApprovalTasks.jsx

```
@@ -0,0 +1,948 @@
+ * Used to build table headers for arrays of JSON objects in the expandable view.
+ * Example:
+ *  [{a:1,b:2},{b:3,c:4}] -> ["a","b","c"]
+ * @param {Array<object>} arr array that contain objects.
+ * @returns {Array<string>} unique keys
+ */
+const getUnionObjectKeys = (arr = []) => {
+    const keys = new Set();
+
+    arr.forEach((obj) => {
+        if (obj && typeof obj === 'object' && !Array.isArray(obj)) {
+            Object.keys(obj).forEach((k) => keys.add(k));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)